libs/ceres/ceres-Natron.patch

```diff
@@ -107,3 +107,14 @@
  namespace ceres {
  namespace internal {
  
+--- CMakeLists.txt.orig	2022-08-08 12:49:31.000000000 -0400
```

Comment:
Change already included in `CMakeLists.txt`.

Reply:
Yes, but this patch file is here to track record of any changes to the upstream code, see README.Natron in each dir.
This makes upgrading much easier